README.md

```diff
@@ -21,15 +21,13 @@ Please visit Minio Docker quickstart guide for more [here](https://docs.minio.io
 ## macOS
 ### Homebrew
 Install minio packages using [Homebrew](http://brew.sh/)
-
 ```sh
 brew install minio/stable/minio
 minio server /data
 ```
 #### Note
-If you previously installed minio using `brew install minio` then reinstall minio from `minio/stable/minio` official repo. Homebrew builds are unstable due to golang 1.8 bugs.
-
-```
+If you previously installed minio using `brew install minio` then it is recommended that you reinstall minio from `minio/stable/minio` our official repo instead.
```

Comment:
Isn't this wording better? 

from `minio/stable/minio` ~our~ official repo instead.

Reply:
we could remove it.. let me do that.